pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go

```diff
@@ -57,10 +57,15 @@ type nodeStatusUpdater struct {
 }
 
 func (nsu *nodeStatusUpdater) UpdateNodeStatuses() error {
+	var abort bool
 	// TODO: investigate right behavior if nodeName is empty
 	// kubernetes/kubernetes/issues/37777
 	nodesToUpdate := nsu.actualStateOfWorld.GetVolumesToReportAttached()
 	for nodeName, attachedVolumes := range nodesToUpdate {
+		if abort {
+			nsu.actualStateOfWorld.SetNodeStatusUpdateNeeded(nodeName)
+			continue
+		}
```

Comment:
On second thought - aggregating error from all nodes may be problematic because we will be constructing these huge error objects which might not be very nice for memory pressure (until they get GCed). So, I would not perhaps use same code. 

But we *do* need to return an error if we can't update nodes.

Reply:
@jingxu97, @jfremy Could someone give their thoughts on the changes suggested by @gnufied. In which direction should we go regarding solving this bug if we have to ?